### PR TITLE
Buttongroup initial

### PR DESCRIPTION
**Pull request by marin-bratanov**

### CODE REVIEW

File: components/buttongroup/overview.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+tags: telerik,blazor,Toggle,button,group
+published: True
+position: 0
+---
+
+# ButtonGroup Overview
+
+This article provides information about the ButtonGroup component and its core features.
+
+The ButtonGroup component is a container for a buttons that can be toggle buttons, and lets you [select one or more]({%slug buttongroup-selection%}), and respond to the [selection and click events]({%slug buttongroup-events%}). The buttons inside fill up the container, match the styling according to the [chosen theme]({%slug general-information/themes%}) and provide the regular button features like images and icons and the other parameters and attributes.
```
Comment (jivanova) [inline components/buttongroup/overview.md:15]:
```suggestion
The ButtonGroup component is a container for buttons that can be toggle buttons, and lets you [select one or more]({%slug buttongroup-selection%}), and respond to the [selection and click events]({%slug buttongroup-events%}). The buttons inside fill up the container, match the styling according to the [chosen theme]({%slug general-information/themes%}) and provide the regular button features like images and icons and the other parameters and attributes.
```
small typo

File: components/buttongroup/overview.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+@code{
+    async Task FirstClick()
+    {
+        Console.WriteLine("the first button was clicked.");
+    }
+
+    async Task SecondClick()
+    {
+        Console.WriteLine("the second button was clicked. It becomes primary when clicked.");
```
Comment (jivanova) [inline components/buttongroup/overview.md:52]:
```suggestion
        Console.WriteLine("the second button was clicked. It becomes selected when clicked.");
```

File: components/buttongroup/overview.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+<TelerikButtonGroup>
+    <ButtonGroupButton OnClick="@FirstClick">First button</ButtonGroupButton>
+    <ButtonGroupToggleButton OnClick="@SecondClick">Second button</ButtonGroupToggleButton>
+</TelerikButtonGroup>
+
+@code{
+    async Task FirstClick()
+    {
+        Console.WriteLine("the first button was clicked.");
```
Comment (jivanova) [inline components/buttongroup/overview.md:47]:
```suggestion
        Console.WriteLine("the first button was clicked.");
```

Should we add more info that the difference between the two types of buttons is the selection capabilities? That ButtonGroupButton is not selectable.

Comment (marin-bratanov) [reply components/buttongroup/overview.md:47]: This is handled in the Selection article and in the SelectedChanged event section, so I don't think we should add it here too.

**Pull request merged**